foodsaving/utils/email_utils.py

```diff
@@ -92,6 +99,107 @@ def prepare_emailinvitation_email(invitation):
     }, to=invitation.email)
 
 
+def calculate_group_summary_dates(group):
```

Comment:
I thought Nick moved that code to `groups/emails.py`? Seems like a merge problem.